.github/workflows/ci.yaml

```diff
@@ -178,12 +178,14 @@ jobs:
       # Setup: rustc
       - if: matrix.install == 'rustup'
         run: |
-          rustup default nightly-${{ matrix.rustc }}
+          rustup default beta-${{ matrix.rustc }}
```

Comment:
Can you keep at least a single nightly job as canary in case rustc changes unstable features or there is a regression?

Reply:
Yeah, that is a good idea. We can do that in a follow-up PR, it is not urgent (i.e. the beta was pretty much nightly when I starting using it :-P).

Reply:
Opened https://github.com/Rust-for-Linux/linux/issues/406.